src/tests/SearchForm.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchForm from "../components/SearchForm";
+
+describe("SearchForm", () => {
+  const validProps = {
+    searchText: "Test Search Text",
+    setSearchText: jest.fn(),
+    onSubmit: jest.fn(),
+  };
+
+  test("renders", () => {
+    const { asFragment } = render(
+      <SearchForm
+        searchText={validProps.searchText}
+        setSearchText={validProps.setSearchText}
+        onSubmit={validProps.onSubmit}
+      />
+    );
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  test("button displays correct text", () => {
+    render(
+      <SearchForm
+        searchText={validProps.searchText}
+        setSearchText={validProps.setSearchText}
+        onSubmit={validProps.onSubmit}
+      />
+    );
+
+    const button = screen.getByText(/Search/i);
```

Comment:
This test is passing, but not for the right reason...
The regex you have given here is equivalent to "find the string 'Search' somewhere on the page, regardless of the case". The actual search text you are expecting to be rendered however is "Search", and so this is matching because 'Search' is in in those valid props. 
To make this test much more useful and accurate, you should search for the exact string you expect to see, in this case you dont need a regex and could just pass "Search" into `getByText()`